tm2/pkg/crypto/secp256k1/bench_test.go

```diff
@@ -9,6 +9,8 @@ import (
 )
 
 func BenchmarkKeyGeneration(b *testing.B) {
+	b.Skip("too slow (more than 30 mins)")
```

Comment:
See https://github.com/gnolang/gno/pull/670

Reply:
Does that mean that it is still too slow?

Reply:
Yeah, it's super slow for even small loop calls (ex. like 100 calls), let alone these benchmark limits